docs/conversion.md

```diff
@@ -15,7 +15,7 @@ This document outlines all the conversion details regarding `docker-compose.yaml
 | depends_on |  | N |  |  |
 | dns |  | N |  |  |
 | dns_search |  | N |  |  |
-| tmpfs |  | N |  |  |
+| tmpfs |  | Y | [Tmpfs](https://kubernetes.io/docs/concepts/storage/volumes/#emptydir) | Creates `emptyDir` volume with `medium:Memory` & mounts given directory inside container | 
```

Comment:
Should be: `Pod.Spec.Containers.Volumes.EmptyDir` for description instead Tmpfs.